src/ClusterBootstrap/cloud_init_aztools.py

```diff
@@ -17,15 +17,22 @@
     remove_nsg_rule_whitelist, \
     delete_nsg_rule_whitelist, \
     create_nsg_rules_with_service_tags, \
+    create_nsg_rule_with_service_tag, \
     delete_nsg_rules_with_service_tags, \
     create_logging_storage_account, \
     create_logging_container, \
     delete_logging_storage_account, \
     get_connection_string_for_logging_storage_account
 
 from cloud_init_deploy import load_node_list_by_role_from_config
+from ctl import run_kubectl
 sys.path.append("../utils")
-from ConfigUtils import add_configs_in_order
+from ConfigUtils import add_configs_in_order, merge_config
+
+ENV_CNF_YAML = 'config.yaml'
+STATUS_YAML = 'status.yaml'
+ACTION_YAML = 'action.yaml'
```

Comment:
If you replace `az_complementary.yaml` with `action.yaml`, you will need to replace it everywhere, especially `src/ClusterManager/test/utils.py`, otherwise @xudifsd won't be happy.

Reply:
sure, I would update all existing yaml files.